src/errors/ActiveNetworkNotFound.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+class ActiveNetworkNotFoundError extends Error {
+  constructor() {
+    const message = 'Active network not found'
```

Comment:
This message is not localized. What use does it have? We could not display it to user.

Reply:
this message is useless and could be easy removed